packages/pages/src/vite-plugin/build/buildStart/buildStart.ts

```diff
@@ -60,18 +61,26 @@ const copyPluginFiles = (fileEmitter: EmitFile) => {
     startLog: "Copying Yext plugin files",
   });
 
-  const currentPath = new URL(import.meta.url).pathname;
-  const pathToPluginsDir = path.resolve(currentPath, "../../../../plugin");
-  const pluginFiles = glob.sync(`${pathToPluginsDir}/*.ts`);
+  const currentPath = fileURLToPath(import.meta.url);
+  const pathToPluginsDir = path.resolve(
+    currentPath,
+    path.join("..", "..", "..", "..", "plugin")
+  );
+  // We must use path.resolve to reconcile filepaths on Windows as glob returns filepaths with forward slashes by default.
+  const pluginFiles = glob
+    .sync(`${pathToPluginsDir}/*.ts`)
```

Comment:
shouldn't this be joined as well? Or is glob.sync posix compliant

Reply:
glob suggests users to use `/` even on windows